### PR TITLE
CC-26202 Added additional validation to form fields.

### PR DESCRIPTION
**Pull request by artem-png**
* Branch: backport/2.16.0
* Ticket: https://spryker.atlassian.net/browse/CC-26202
* Version: 2.16.0

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   ProductMerchantPortalGui  | patch                 |                       |

-----------------------------------------

##### Change log

Fixes

- Adjusted `ProductLocalizedAttributesForm` to sanitize `name` and `description` fields.
- Impacted `UpdateProductConcreteController::indexAction()` and `UpdateProductAbstractController::indexAction()` with these changes.

### CODE REVIEW

File: src/Spryker/Zed/ProductMerchantPortalGui/Communication/Form/ProductLocalizedAttributesForm.php [hunk 1]
```diff
@@ -139,6 +139,8 @@ protected function addNameField(FormBuilderInterface $builder)
                     'max' => 255,
                 ]),
             ],
+            'sanitize_xss' => true,
```
Comment (DmitryLymarenko) [inline src/Spryker/Zed/ProductMerchantPortalGui/Communication/Form/ProductLocalizedAttributesForm.php:142]: this functionality is available only in GUI 3.48

**Review by DmitryLymarenko — CHANGES_REQUESTED** [review]
currently release type is a patch, but after composer adjustment it will become minor

File: src/Spryker/Zed/ProductMerchantPortalGui/Communication/Form/ProductLocalizedAttributesForm.php (hunk 1 above)
Comment (artem-png) [reply src/Spryker/Zed/ProductMerchantPortalGui/Communication/Form/ProductLocalizedAttributesForm.php:142]: 2.15.0 version has 3.48 GUI, so all good

Comment (DmitryLymarenko) [reply src/Spryker/Zed/ProductMerchantPortalGui/Communication/Form/ProductLocalizedAttributesForm.php:142]: I see
Roman introduced another backport recently

**Review by DmitryLymarenko — COMMENTED** [review]
currently branching is wrong
it's aimed for a minor, but we need a patch

**Pull request closed without merging**